content/docs/networking/load-balancing.md

```diff
@@ -1,47 +1,99 @@
+</details>
+
+### Semantic Addressing
+
+The semantic addressing happens on the above mentioned service layer.
+Similar to a cluster IP in Kubernetes, these addresses reference all the instances (replicas) of a microservice with a single address. This address does not change when scaling up the instances or when migrating them.
+Anyway, unlike the Kubernetes cluster IP, when deploying a service in Oakestra, the platform provides as many Service IP addresses as the number of balancing policies supported (and active) within the platform.
+
+#### Example
+
+[NetExample](/network/NetExample_dual_stack.png)
+
+A developer can communicate with any instance of Service B either with **Round Robin** balancing policy or the **Closest** balancing policy. The former balances the traffic evenly between all the instances, and the latter finds the geographically closer instance.
+
+**Reminder**: Currently the **IPv6 Closest** balancing strategy IP addresses (reserved, but not implemented) are in the `fdff:1000::/21` subnet, and **IPv6 Round Robin** in `fdff:2000::/21`. In IPv4, both balancing strategies take their IPs from the `10.30.0.0/16` subnet. For further information, take a look at the [IPv4](/networking/ipv4-address-space.md) or [IPv6](/networking/ipv6-address-space.md) address space documentation pages.
 
-One might ask, **what's the difference between an Instance IP and a Namespace IP?** They operate on 2 different abstraction layers. The Namespace IP depends on the virtualized subnetwork of a worker node and changes when migrating an application from one node to another. It cannot be provisioned beforehand and must not be used by developers. The Instance IP is part of the Service layer abstraction. Therefore it identifies an instance regardless of migration operations, scales up, scales down, and can be provisioned even before the deployment of a service instance.  
+Service A performs the first request <img src="/network/NetArchExample_envelope_1.png" alt= "envelope_1" width="30"> using the ServiceIP <font style="color:red">fdff:1000::1</font> representing the closest instance balancing policy.
+The network components' proxy converts the address to the Namespace IP of Service B Instance 1, which looks like it is the geographically closer service. The message will be, therefore **transparently** delivered to the closest instance of Service B. Note that the Namespace IP is the real address of the instance, the one provisioned at deployment time, unique and dynamic. An application developer never sees or uses this address, as it depends on the subnetwork of the specific machine where the service is deployed. Find out more about the Namespace IP by reading the implementation details.
```

Comment:
"Find out more about the Namespace IP by reading the implementation details." -> where? This either needs a link or should be removed.

Reply:
Removed it. We clarify everything needed in this document.